overlay_gl/CMakeLists.txt

```diff
@@ -91,6 +91,8 @@ if(NOT APPLE)
 					"-ldl"
 					"-lrt"
 			)
+      # install 32bit overlay library
+      install(TARGETS overlay_gl_x86 LIBRARY DESTINATION "${MUMBLE_INSTALL_LIBDIR}")
 		endif()
```

Comment:
The indentation seems to be off (we indent using tabs)

Reply:
Ah, my bad. This is fixed now.

Reply:
No it's not. It is still indented using spaces xD

Reply:
Huh? The changes must have gotten lost somehow when I split the commit in two.
Third times the charm :crossed_fingers:

Reply:
Yes indeed :D